### PR TITLE
Added additional options for make release docs

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*
* Make it easier to generate doc for different use cases

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cmd/release/docs/main.go
Comment (jaxesn) [inline cmd/release/docs/main.go:130]: Intentional?

Comment (kschumy) [reply cmd/release/docs/main.go:130]: yeah this was a bug. I had [fixed this issue with the other flags](https://github.com/aws/eks-distro/pull/493/files#diff-c11fdc4773cfc3c3984e5c0cb9093a6bd9d8fa375aa69c3d621ce45140a947ffL23-R65) awhile ago, but somehow this one slipped through

**Review by jaxesn — APPROVED** [review]
/ok-to-test
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-distro/pull/566#pullrequestreview-763550821" title="LGTM">jaxesn</a>*, *<a href="https://github.com/aws/eks-distro/pull/566#" title="Author self-approved">kschumy</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-distro).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-distro/blob/main/OWNERS)~~ [jaxesn,kschumy]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
